tests/device_tests/test_msg_backup_device.py

```
@@ -292,13 +292,15 @@ def test_backup_slip39_advanced_custom(
+    assert [len(group) for group in IF.mnemonics] == [
+        count for _threshold, count in groups
+    ]
 
     selected_mnemonics: list[str] = []
-    offset = 0
-    for member_threshold, member_count in groups[:group_threshold]:
-        selected_mnemonics.extend(IF.mnemonics[offset : offset + member_threshold])
-        offset += member_count
+    for (member_threshold, _member_count), group_mnemonics in zip(
+        groups[:group_threshold], IF.mnemonics
+    ):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)